### ISSUE TITLE
lazlo askme module responding in different channels

### ISSUE THREAD

**Issue by mbbroberg** [djosephsen/lazlo#6]
It seems channels are blocking each other's response. The behavior: I'm chatting with lazlo as "banterbot" via DM and on a channel called "bot-spam": 

In the channel: 

```
Matthew Brender [10:42 PM] 
banterbot askme shalom

[10:42] 
banterbot?

[10:42] 
banterbot help

[10:42] 
banterbot askme

[10:42] 
banterbot ping

banterbotBOT [10:43 PM] 
mjbrender: yup. still here.

Matthew Brender [10:43 PM] 
banterbot askme hello

[10:44] 
banterbot askme hello
```

In DM: 

```

You answered: 'banterbot askme hello'

[10:42] 
hello

Matthew Brender [10:42 PM] 
banterbot askme hi

banterbotBOT [10:42 PM] 
You answered: 'banterbot askme hi'

[10:42] 
hi

[10:42] 
########## Modules In use: 
"banterbot linkme foo" : creates a clickable link at servername/foo
"banterbot brain [set|get] <key> <value>": tests lazlo's persistent storage (aka the brain)
"banterbot help": prints the usage information of every registered plugin
"banterbot askme foo" : replies with the question: foo?
banterbot qtest : runs an automated question test
"banterbot (ping|syn)" : Test that the bot is currently running

Matthew Brender [10:43 PM] 
banterbot askme nothing

banterbotBOT [10:43 PM] 
You answered: 'banterbot askme nothing'

[10:43] 
shalom

Matthew Brender [10:44 PM] 
banterbot askme something

banterbotBOT [10:44 PM] 
You answered: 'banterbot askme something'

[10:44] 
hello
```

**Comment by mbbroberg** [comment]
TL;DR -- a response from a new question in the original channel will result in multiple answers from the backlog. Thought it was cool enough to share :+1:
